server/app/api/plugin_management.py

```
@@ -0,0 +1,105 @@
+from fastapi import Request
+
+
+router = APIRouter()
+
+
+def _pm(request: Request):
+    return request.app.state.plugin_manager
+
+
+@router.get("/plugins")
+def list_plugins(pm=Depends(_pm)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)